### DISCUSSION TITLE
Change chapter heading spacing without changing title heading spacing

### DISCUSSION THREAD

**Discussion by anton-granzow** [tud-cd/tudscr#95] [Q&A] [answered]
Hey, is there a way to change the spacing of chapter headings without changing the titlepage headings? If I set `headingsvskip` to something like `-80pt` the title heading is way to high but the chapter heading is right how I want it.

**Comment by mrpiggi** [comment]
Well, just adapt the height again after the title. Or you could use `pageheadingsvskip` additionally, if you are not using `chapterpage=true`

**Comment by anton-granzow** [comment]
> Well, just adapt the height again after the title.

How do I do this? I tried renewcommand bit it didnt work. Maybe I did something wrong? Sorry if it is a dumb question, I'm not very experienced with more advanced Latex.

**Comment by mrpiggi** [comment, marked as answer]
You could use both **options** either within the optional argument of `\documentclass[pageheadingsvskip=...,headingsvskip=...]{tudscr...]` or just use `\TUDoptions{pageheadingsvskip=...,headingsvskip=...}` anywhere in the document
